requirements/edx-sandbox/post.txt

```diff
@@ -6,5 +6,6 @@
 
 # Packages to install in the Python sandbox for secured execution.
 scipy==0.14.0
-lxml==3.4.4
+# lxml is also in requirements/edx/base.txt
+lxml==3.8.0
 matplotlib==1.3.1
```

Comment:
This library doesn't seem to be used by any code that lives in edx-platform. If it's used by an XBlock, why is it only in sandbox requirements?

Reply:
Both excellent questions! @edx/devops Can anyone on your team shed light on the origin/purpose of the edx-sandbox requirements? Is it a codejail origin?